### PR TITLE
Document the release procedure

### PR DESCRIPTION
**Pull request by glasserc**
I've found instructions like these incredibly helpful in Kinto-related projects. This is just the set of steps I understood from Mathieu in chat this morning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,16 @@
 ## Licence
 
 Apache 2
+
+## Releasing
+
+We don't use `zest.releaser` right now because of some problems with
+releasing a package that is not at the root of the repo (`jobs/`), but
+this could change if we figure out how.
```
Comment (leplatrem) [inline README.md]: I think it's also because we don't really need pypi. It runs in a lambda that we publish as a zip on the release page

Comment (glasserc) [reply README.md]: Ah right, I forgot to mention about the release page, thanks.

**Comment by peterbe** [comment]
It's been a while since I wrote [this](https://github.com/mozilla-services/tecken/blob/master/bin/make-tag.py) but it's really served me well. Whenever I want to upgrade Stage I just type `./bin/make-tag.py` and hit Enter. (CloudOps have it set up to upgrade Stage immediately when a new release tag is created)

It's good to have instructions but it's sometimes even better to have it all automated away.

**Comment by peterbe** [comment]
What I'm suggesting is that we look at that `make-tag.py` and copy it into the buildhub repo and rewrite the parts that need to be different. Then we can extend it with some of the stuff about filing bugs in bugzilla. That too can easily be automated.

**Comment by glasserc** [comment]
I agree that it's valuable to have these processes automated. Let's look into what it would take to get zest.releaser working on this project, and extending it with hooks to do the Github/Bugzilla bits. I've opened #346 about this.

File: README.md [hunk 2]
```diff
@@ -10,3 +10,19 @@
...
+
+We don't use `zest.releaser` right now because of some problems with
+releasing a package that is not at the root of the repo (`jobs/`), and
+because we have no interest in uploading this project to PyPI, but
+this could change if we figure out how.
+
+The current procedure is:
+
+* Bump version in `jobs/setup.py`
+* `git tag x.y.z`
```
Comment (Natim) [inline README.md]: What about updating the release date in the `CHANGELOG.rst` and running `make build-dependencies`

File: README.md [hunk 3]
```diff
@@ -10,3 +10,24 @@
...
+* Bump version in `jobs/setup.py`
+* Update the release date in `jobs/CHANGELOG.rst`
+* `make build-dependencies`
+* `git commit -am "Bump x.y.z"`
+* Open PR, wait for it to become green
+* Merge PR
+* `git tag x.y.z`
+* `git push --tags origin`
+* `make lambda.zip`
+* Add a release on Github with the lambda.zip attached
```
Comment (peterbe) [inline README.md]: What does this mean? Going to https://github.com/mozilla-services/buildhub/releases/new and manually upload the `lambda.zip` file with the browser?

Comment (glasserc) [reply README.md]: Based on previous releases, I think so, but I've never done it myself.

File: README.md [hunk 4]
```diff
@@ -10,3 +10,24 @@
...
+* Open a ticket like https://bugzilla.mozilla.org/show_bug.cgi?id=1426340 to get it deployed
```
Comment (peterbe) [inline README.md]: So, the only variable is the tag name? 
A link to create the bug can be made with this as a query string variable. 

See how the "Bugzilla" tab on Crash-stats have links to enter a new bug with lots of the stuff already filled in. 
https://crash-stats.mozilla.com/report/index/19ef17b4-3864-4a55-b9a6-ae6e30180204#tab-bugzilla

File: README.md (hunk 3 above)
Comment (leplatrem) [reply README.md]: Yes, manual upload so far.

**Pull request merged**